cli/gluster-block.c

```diff
@@ -788,6 +798,7 @@ glusterBlockReplace(int argcount, char **options, int json)
     LOG("cli", GB_LOG_ERROR, "failed replace on volume %s",
         robj.volume);
   }
+  GB_FREE(robj.cmd);
```

Comment:
We will leak cmd at line 790. Why not just move these GB_FREE() to out label every where possible. If pointer is NULL, GB_FREE just skips freeing. See implementation for details.